Dockerfile

```diff
@@ -82,13 +82,15 @@ RUN INSTALL_MSG=$(/bin/bash /tmp/torchaudio_build/scripts/installers/install_tor
 
 # install nemo dependencies
 WORKDIR /tmp/nemo
+ENV LHOTSE_REQUIRE_TORCHAUDIO=0
```

Comment:
Is this needed?

Reply:
Yes, it prevents Lhotse from pulling/checking torchaudio.

Reply:
Does the user need to set it when not using the dockers?

Reply:
Only if they can't have torchaudio installed for some reason. IIUC the issue with torchaudio mostly stems for upstream docker images not having it pre-built and the need to build from source when building NeMo containers (which may fail for a number of reasons). But outside these docker images it's straightforward to pip/conda install torchaudio alongside torch. WDYT?